Meta data bulk edit

MarvinPerry opened the pull request:
Adds a script for generating a csv of public open data items that can be edited and updated in bulk.

MarvinPerry committed:
working rough first pass of xml generator

MarvinPerry committed:
cleans code a litle and is just before major changes

MarvinPerry committed:
right before major overhaul to generated xml

MarvinPerry committed:
first pass with full functionality enabled

MarvinPerry committed:
updates README.md with description and images

MarvinPerry committed:
cleans up generateSeedXml function

MarvinPerry committed:
improves metadata handling so changes are reflected on item page

MarvinPerry committed:
fixes broken links and corrects formatting in README.md

MarvinPerry committed:
corrects incorrect file in documentation and typos

brooksjbr commented:
@MarvinPerry - An enhancement to the script would be to print out a status to the user when generating the CSV. The script is taking a pretty long time, it would be useful to get messaging that the script is still working.

MarvinPerry committed:
fixes missing directory error

MarvinPerry committed:
addresses bug where some items would cause the script to break

MarvinPerry commented:
@brooksjbr I addressed the breaking issues we found, and added a message that it could take a while initially, based on the number of items in the org.

MarvinPerry committed:
corrects error handling

MarvinPerry committed:
corrects error in loop

MarvinPerry committed:
corrects the readme to reflect the repo being moved

MarvinPerry committed:
changes structure of repo for readability

MarvinPerry committed:
corrects readme file name

MarvinPerry committed:
updates csv fields and improves documentation

MarvinPerry committed:
fixes error message bug and updates readme with cinda update instructions

MarvinPerry committed:
changes loop

MarvinPerry committed:
Update README.md

MarvinPerry committed:
adds notes about table in readme

MarvinPerry committed:
Update README.md

updated the table for accuracy

MarvinPerry committed:
Merge branch 'metaDataBulkEdit' of github.com:Esri/arcgis-admin-python-scripts into metaDataBulkEdit

MarvinPerry committed:
adds revised fields for csv

MarvinPerry committed:
resolves issues caused by invalid metadata
